iothub/service/src/ImportExportJobParameters.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Input parameters for jobs
+    /// </summary>
+    public class ImportExportJobParameters
```

Comment:
Should it instead be called ImportExportJobInputParameters or ImportExportJobRequestParameters

Reply:
I find the extra words "input" and "request" a bit redundant, don't you? Parameters are things one passes as input or to a request.

Reply:
I should note that I originally called this JobPropertiesParameters, but after looking at more of our source code we have several different ambiguously named Job* classes for jobs other than import and export. I figured I'd disambiguate here. I don't take naming lightly, and do welcome feedback on the name.

At the same time, we should generally strive for concisely named types that clearly convey their identity. Obviously those two things are a trade-off.